hazelcast/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -87,7 +88,8 @@
  *
  * <h3>How the methods are called</h3>
  * <p>
- * Besides {@link #init}, {@link #close} and {@link #isCooperative} the methods
+ * Besides {@link #init}, {@link #close}, {@link #isCooperative} and
```

Comment:
You should reflow the text to 72 characters, check out this plugin: https://plugins.jetbrains.com/plugin/7234-wrap-to-column

Reply:
Rest of the comments in this file are wrapped to 80 characters and sometimes even more. Should I reformat entire file or wrap to 80 characters?

Reply:
done. wrapped to 80 characters to be consistent with rest of the file